api/Document.json

```diff
@@ -8171,16 +8179,22 @@
           "support": {
             "chrome": [
               {
-                "version_added": "33"
+                "version_added": "62"
```

Comment:
33 vs. 62 is a big difference. Can you check where 33 came from?

Reply:
The "33" had come from the initial wiki migration.  I did a quick test to confirm the results:

```js
document.onvisibilitychange = function() {
	console.log("Page visibility changed!");
}
```

Reply:
Thanks for double checking!